src/axom/quest/tests/quest_all_nearest_neighbors.cpp

```diff
@@ -168,7 +168,7 @@ TEST(quest_all_nearnbr, cplx_13region_query)
 {
   SLIC_INFO("*** 13-region closely-packed query.");
 
-  // Pre-formatted lines, not to be disturbed by uncrustify: *INDENT-OFF*
+  // Pre-formatted lines, not to be disturbed by clang format: clang-format off
```

Comment:
I say just go to the normal comment and be consistent with the rest of the code.

Reply:
Good suggestion.  I removed the stuff before `clang-format off`, as suggested.